### PR TITLE
Translations: translate timed exam timer on scroll message

### PR DESCRIPTION
**Pull request by Salomari1987**
### Description

TASK: [Translate the timed exam feature](https://app.asana.com/0/inbox/198215864990332/157680109305774/198692551567628)

### Notes

- Translating the message displayed on the horizontal timer bar on top
- Changing the github requirements of the repo to point to Edraak's fork

### Testing
- [ ] i18n
- [ ] RTL

### CODE REVIEW

**Comment by Salomari1987** [comment]
Update: translate timed exam in navigation pane from python

**Comment by OmarIthawi** [comment]
I don't think I can review this on time @Salomari1987, please feel free to merge if it passes QA.

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -94,7 +94,8 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 -e git+https://github.com/edx/edx-reverification-block.git@0.0.5#egg=edx-reverification-block==0.0.5
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 git+https://github.com/edx/edx-organizations.git@release-2015-12-08#egg=edx-organizations==0.2.0
-git+https://github.com/edx/edx-proctoring.git@0.11.6#egg=edx-proctoring==0.11.6
+-e git+https://github.com/Edraak/edx-proctoring.git@edraak-0.11.6#egg=edx-proctoring==edraak-0.11.6
```
Comment (OmarIthawi) [inline requirements/edx/github.txt]: Looks good. Just use the tag `edraak/0.11.6.1` instead.

**Comment by OmarIthawi** [comment]
Didn't test it yet, but it passes QA.

**Pull request merged**